RadarSDK/RadarLocationManager.m

```diff
@@ -372,33 +401,53 @@ - (void)replaceSyncedGeofences:(NSArray<RadarGeofence *> *)geofences {
     }
 }
 
-- (void)replaceBubbleGeofence:(CLLocation *)location radius:(int)radius {
-    [self removeBubbleGeofence];
-
-    NSString *identifier = [NSString stringWithFormat:@"%@_%@", kRegionIdentifer, [[NSUUID UUID] UUIDString]];
-    CLRegion *region = [[CLCircularRegion alloc] initWithCenter:location.coordinate radius:radius identifier:identifier];
-    [self.locationManager startMonitoringForRegion:region];
-}
-
 - (void)removeSyncedGeofences {
     for (CLRegion *region in self.locationManager.monitoredRegions) {
-        if ([region.identifier hasPrefix:kRegionSyncIdentifer]) {
+        if ([region.identifier hasPrefix:kSyncGeofenceIdentifierPrefix]) {
             [self.locationManager stopMonitoringForRegion:region];
         }
     }
 }
 
-- (void)removeBubbleGeofence {
+- (void)replaceSyncedBeacons:(NSArray<RadarBeacon *> *)beacons {
+    [self removeSyncedBeacons];
+
+    BOOL tracking = [RadarSettings tracking];
+    RadarTrackingOptions *options = [RadarSettings trackingOptions];
+    if (!tracking || !options.beacons || !beacons) {
+        return;
+    }
+
+    NSUInteger numBeacons = MIN(beacons.count, 9);
```

Comment:
Not a big deal, but the `searchBeacons` call is passing `limit = 10`, but we're capping it at 9 here.

Reply:
I can switch if it reduces server load. We get 20 slots on iOS, need to save one for the bubble geofence, then split the rest between synced geofences and synced beacons

Reply:
Probably worth a comment here explaining why it's being limited and where `9` comes from.